src/app/admin/time-table/time-table-add-dialog/time-table-add-dialog.component.html

```diff
@@ -10,15 +10,38 @@ <h3 mat-dialog-title>{{data.description.title}}</h3>
       </mat-select>
     </mat-form-field>
   </div>
+  <div>
+    <mat-form-field class="example-full-width">
+      <mat-label>Предмет</mat-label>
+      <mat-select [(ngModel)]="data.data.subject_id" name="subject_id" #subject="ngModel">
+        <mat-option *ngFor="let subject of subjects" [value]="subject.subject_id">
+          {{subject.subject_name}}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+  </div>
   <div>
     <mat-form-field>
-      <input matInput [matDatepicker]="picker" placeholder="Виберіть дату" disabled>
+      <input matInput [matDatepicker]="picker"
+             placeholder="Виберіть дату"
+             name="start_date" ngModel
+             #start_date="ngModel"
```

Comment:
Are you sure that we need start_date ref? Why do we need `ngModel` without binding?

Reply:
I've removed unused ref and pushed commit changes to this branch